packages/orm/src/EntityPersister.ts

```diff
@@ -104,7 +104,7 @@ async function batchUpdate(knex: Knex, tx: Transaction, meta: EntityMetadata<any
       UPDATE ${meta.tableName}
       SET ${columns
         .filter((c) => c.columnName !== "id")
-        .map((c) => `"${c.columnName}" = data.${c.columnName}`)
+        .map((c) => `"${c.columnName}" = data."${c.columnName}"`)
```

Comment:
Ah damn, for some reason I thought to put the quotes on the left side but forgot the right side. Thanks!